torchci/lib/bot/cliParser.ts

```diff
@@ -45,7 +45,7 @@ mergeOption.add_argument("-f", "--force", {
   help: `Merge without checking anything. This requires a reason for auditting purpose, for example:
 @pytorchbot merge -f 'Minor update to fix lint. Expecting all PR tests to pass'`,
 });
-mergeOption.add_argument("-ic", "--ignore-current", {
+mergeOption.add_argument("-i", "--ignore-current", {
```

Comment:
I'd also suggest making the old "-ic" parameter continue working for backwards compat reasons (for the folks who've built up muscle memory on the old one).

Alternatively, when someone enters "-ic" reply back with a message saying "'-ic' is deprecated, please use '-i' instead of the same effect"

Reply:
did option 2